seawatch_registration/tests/views/test_add_document_view.py

```diff
@@ -0,0 +1,98 @@
+import tempfile
+from datetime import date
+from django.contrib.auth.models import User
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.test import TestCase, override_settings, Client
+from django.urls import reverse
+
+from seawatch_registration.models import Profile, DocumentType, Document
+from seawatch_registration.tests.views import util
+
+
+class TestAddDocumentView(TestCase):
+
+    def setUp(self) -> None:
+        self.client = Client()
+        self.username = 'testuser1'
+        self.password = '1X<ISRUkw+tuK'
+        self.user = User.objects.create_user(username=self.username, password=self.password)
+        self.user.save()
+        self.url_add_document = reverse('add_document')
```

Comment:
Kann der Teil nicht auch noch in tests/views/util.py ausgelagert werden?

Reply:
ich bin mir nicht sicher wie ich das am besten machen würde und wie viel es am ende sparen würde.